pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: easy_presentation
-description: To present data in cover detail section
-version: 1.0.0
+description: This package present your data in the form of clean presentation view
```

Comment:
This package presents your data in the form of clean presentation view, it also supports simple text, markdown, youtube videos and tabular data form.